### DISCUSSION TITLE
Some ideas to redesign resources

### DISCUSSION THREAD

**Discussion by alvleont** [filamentphp/filament#3859] [Ideas]
Hi!

These are some suggestions to improve the appearance of pages.

- Remove the gray border of cards.
- Add the Icon of the resource into the header.
- Change the "New XXX" button for a Plus (this avoids a bug that happens in spanish when items are written with Female words but "Nuevo" is male). Other option is to change Spanish translation to Create (Crear) or Add (Agregar).

**CURRENT:**
<img width="1385" alt="Captura de Pantalla 2022-09-04 a la(s) 5 15 19 p m" src="https://user-images.githubusercontent.com/47951932/188335449-128e600b-fa1c-480b-b035-05773d478047.png">

**PROPOSAL**
<img width="1378" alt="Captura de Pantalla 2022-09-04 a la(s) 5 21 31 p m" src="https://user-images.githubusercontent.com/47951932/188335650-22fbd7d5-8958-4ebd-8279-20cda6accfe8.png">

**Comment by danharrin** [comment]
We won't be changing the button. You can override that label yourself using `CreateAction::make()->label('Nueva XXX')`
